src/Models/OrganizationProfile.php

```diff
@@ -0,0 +1,31 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Blumilk\Meetup\Core\Models;
+
+use Illuminate\Database\Eloquent\Factories\HasFactory;
+use Illuminate\Database\Eloquent\Relations\BelongsTo;
+
+/**
+ * @property int $id
+ * @property string $link
+ * @property string $label
+ * @property string $icon
+ * @property-read Organization $organization
```

Comment:
```suggestion
 * @property-read int $organizationId
 * @property-read Organization $organization
```

It's a good thing to add keys too.